### PR TITLE
Cardano Shelley Update 3/3

### PR DESCRIPTION
**Pull request by gabrielKerekes**
Add certificates, withdrawals and metadata to CardanoSignTransaction.

### CODE REVIEW

File: docs/methods/cardanoSignTransaction.md [hunk 1]
```diff
@@ -17,9 +17,12 @@ TrezorConnect.cardanoSignTransaction(params).then(function(result) {
...
-* `outputs` - *obligatory* `Array` of [CardanoOutput](../../src/js/types/networks/cardano.js#L69)
+###### [flowtype](../../src/js/types/networks/cardano.js#L62-L109)
+* `inputs` — *obligatory* `Array` of [CardanoInput](../../src/js/types/networks/cardano.js#L71)
+* `outputs` - *obligatory* `Array` of [CardanoOutput](../../src/js/types/networks/cardano.js#L76)
+* `certificates` - *optional* `Array` of [CardanoCertificate](../../src/js/types/networks/cardano.js#L83)
+* `withdrawals` - *optional* `Array` of [CardanoWithdrawal](../../src/js/types/networks/cardano.js#L88)
+* `metadata` - *optional* `String`
 * `fee` - *obligatory* `String`
 * `ttl` - *obligatory* `String`
 * `protocolMagic` - *obligatory* `Integer` 764824073 for Mainnet, 42 for Testnet
```
Comment (janmazak) [inline docs/methods/cardanoSignTransaction.md]: I would reorder this, with obligatory stuff first, and elements in the same order as they are serialized in tx body.
(Affects many places, so maybe not really worth it, but any comparison against specs / expected values is made harder by this non-standard ordering.)

**Review by janmazak — COMMENTED** [review]
Maybe someone from Adalite should take a look at the API they will integrate with.

File: docs/methods/cardanoSignTransaction.md (hunk 1 above)
Comment (gabrielKerekes) [reply docs/methods/cardanoSignTransaction.md]: I did reorder them [here](https://github.com/vacuumlabs/connect/pull/3/commits/c9dcfc1e00cd23f65fc8dfbc541fe1ef3eca017e). In the readme I put the obligatory properties first, so `protocolMagic` and `networkId` are mixed into the transaction elements, but I think it's better this way. In all other places `protocolMagic` and `networkId` are last.

**Comment by refi93** [comment]
closed in upstream repo already

**Pull request closed without merging**